fitnessllm_dataplatform/stream/strava/etl_utils.py

```
@@ -0,0 +1,243 @@
+            destination=f"dev_strava.{stream}",
+            job_config=bigquery.LoadJobConfig(
+                write_disposition=bigquery.WriteDisposition.WRITE_APPEND
+            ),
+        )
+        insert_metrics(
+            metrics_list=metrics,
+            destination="dev_metrics.metrics",
+            timestamp=timestamp,
+            status=Status.SUCCESS,
+        )
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)